lib/site.jetpack-module.js

```diff
@@ -0,0 +1,75 @@
+
+/**
+ * SiteJetPackModule methods
+ *
+ * @param {String} sid - site id
+ * @param {WPCOM} wpcom - wpcom instance
+ * @return {Null} null
+ */
+class SiteJetPackModule {
+	constructor( id, sid, wpcom ) {
+		if ( ! sid ) {
+			throw new Error( '`site id` is not correctly defined' );
+		}
+
+		if ( ! ( this instanceof SiteJetPackModule ) ) {
+			return new SiteJetPackModule( id, sid, wpcom );
+		}
+
+		this.wpcom = wpcom;
+		this._id = id;
+		this._sid = sid;
+		this.path = `/sites/${this._sid}/jetpack/modules`;
```

Comment:
I've been working on this to try to tackle to these inconsistencies. The idea is create a `SubClass` and extend Site class from this one. https://github.com/Automattic/wpcom.js/pull/185.
So I think we can keep this code here and then work in the `SubClass` implementation.